### PR TITLE
Move requirements to requirements.txt

### PR DESCRIPTION
**Pull request by jeremycline**
This makes it easy to `pip install -r` or use Ansible's pip module to install requirements.

Signed-off-by: Jeremy Cline jeremy@jcline.org

### CODE REVIEW

**Comment by pypingou** [comment]
:+1: for me

**Comment by jeremycline** [comment]
There was a missing dependency on `html5lib` which in turn requires a version of setuptools newer than the version installed on the Travis test runner by default, so I've updated a few more things. Tests should pass in Travis now.

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+beautifulsoup4
+docutils
+fmn.rules
+html5lib
```
Comment (pypingou) [inline requirements.txt:7]: Since this one wasn't in the original list, should we document why we added it?

**Comment by pypingou** [comment]
One small comment otherwise :+1: 

Feel free to merge (or wait for travis) w/ or w/o the fix for the question asked, I don't consider it a blocker, more a nice to have

File: requirements.txt (hunk 1 above)
Comment (jeremycline) [reply requirements.txt:7]: I added a comment and filed https://github.com/fedora-infra/fmn.lib/issues/64 so we can eventually remove the dependency

**Pull request merged**